src/components/ErrorPage/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+
+import StatusAlert from '../StatusAlert';
+
+const NotFoundPage = () => (
+  <div className="container mt-3">
+    <Helmet>
+      <title>Page Not Found</title>
+    </Helmet>
+    <div className="text-center py-5">
+      <h1>404</h1>
+      <p className="lead">Oops, sorry we can&apos;t find that page!</p>
+      <p>Either something went wrong or the page doesn&apos;t exist anymore.</p>
+    </div>
+  </div>
+);
+
+const ErrorPage = (props) => {
+  const errorMessage = props.message || 'An unknown error has occured.';
+
+  return (
+    <div className="container">
+      {props.status === 404 ? (
```

Comment:
We may want to make a specific error page for 500 errors so the user knows it on our end not theirs. Fine with this being in a followup though.

Reply:
good suggestion, will create a story for it for future